src/models/PoolLiquidity.ts

```diff
@@ -0,0 +1,33 @@
+import {index, modelOptions, prop, Severity} from '@typegoose/typegoose';
+
+@index({address: 1})
+@index({address: 1, chainId: 1, protocol: 1, token0: 1, token1: 1})
+@modelOptions({options: {allowMixed: Severity.ALLOW}})
+export class PoolLiquidity {
```

Comment:
let's use same name as for pool scanner PR, because this model will be the same for scanner and for liquidity